src/doc/python_scripting/functions.rst

```diff
@@ -1510,38 +1510,46 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineArrayExpression function returns 1 on success and 0 on failure.
 
 
 **Description:**
 
     DefineArrayExpression creates new array variables.
-    Expression variables can be plotted like any other variable.
-    The variableName argument is a string that contains the name of the new
-    variable. You can pass the name of an existing expression if you want
-    to provide a new expression definition.
-    The expression argument is a string that contains the definition of the
-    new variable in terms of math operators and pre-existing variable names
-    Reference the VisIt User's Manual if you want more information on
-    creating expressions, such as expression syntax, or a list of built-in
-    expression functions.
+    Array variables are a collection of scalar variables that are grouped together.
+    Currently there are no plots that take array variables, but they can be used in queries, such as the "XRay Image" query or used as a pick variable that creates a Histogram plot of the array variable for a specific zone.
+
+    The variableName argument is a string that contains the name of the new variable.
+    You can pass the name of an existing expression if you want to provide a new expression definition.
+    The expression argument is a string that contains the definition of the new variable in terms of `built-in expressions <../using_visit/Quantitative/Expressions.html#built-in-expressions>`_ and pre-existing variable names using VisIt_'s `expression grammar <../using_visit/Quantitative/Expressions.html#expression-grammar>`_.
```

Comment:
If you wanna do the link to expression grammar section this way, I don't think you should be using `.html` extension. And, believe you can abbreviate this to...

```
`Expression grammar`_
```

Just beware that the resulting link will break if the section name is changed. One good reason for using `:ref:<...>` (which comes with the added burden of defining the labels/anchors) is link-checking and warning logic

Reply:
Changed them all to :ref:.